app/uk/gov/hmrc/cataloguefrontend/connector/PrCommenterConnector.scala

```diff
@@ -38,7 +39,7 @@ object PrCommenterConnector {
     created  : Instant
   ) {
     // Repos with joint ownership of 8+ teams, defines the repo as shared
-    def isShared: Boolean = teamNames.length >= 8
+    def isShared: Boolean = teamNames.length >= Constant.sharedRepoTeamsCutOff
```

Comment:
Constant could be moved to `GitRepository` object - since that seems to own the value. We're just trying to keep the Report consistent.